src/main/java/it/gov/pagopa/spontaneouspayment/mapper/ConvertOrganizationModelToOrganizationEntity.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.spontaneouspayment.mapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.modelmapper.Converter;
+import org.modelmapper.spi.MappingContext;
+
+import it.gov.pagopa.spontaneouspayment.entity.Organization;
+import it.gov.pagopa.spontaneouspayment.entity.ServiceRef;
+import it.gov.pagopa.spontaneouspayment.model.CreateEnrollmentModel;
+import it.gov.pagopa.spontaneouspayment.model.OrganizationEnrollmentModel;
+
+public class ConvertOrganizationModelToOrganizationEntity
+		implements Converter<OrganizationEnrollmentModel, Organization> {
+
+	@Override
+	public Organization convert(MappingContext<OrganizationEnrollmentModel, Organization> context) {
+		OrganizationEnrollmentModel source = context.getSource();
+		List<ServiceRef> enrollments = new ArrayList<>();
+		if (null != source.getEnrollments() && !source.getEnrollments().isEmpty()) {
+			for (CreateEnrollmentModel cem: source.getEnrollments()) {
```

Comment:
I suggest to use model mapper without manual converter. If you use the same names for the properties, [Model mapper](http://modelmapper.org/) automatically determining how one object model maps to another

Reply:
Use explicit converter only for complex mapping